mesheryctl/pkg/utils/platform_test.go

```diff
@@ -0,0 +1,643 @@
+package utils
+
+import (
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"runtime"
+	"testing"
+
+	"github.com/jarcoal/httpmock"
+	"github.com/layer5io/meshery/mesheryctl/internal/cli/root/config"
+	"github.com/layer5io/meshery/mesheryctl/pkg/constants"
+	meshkitkube "github.com/layer5io/meshkit/utils/kubernetes"
+	log "github.com/sirupsen/logrus"
+	"github.com/spf13/viper"
+)
+
+func init() {
+	// Get current directory
+	_, filename, _, ok := runtime.Caller(0)
+	if !ok {
+		log.Fatal("Not able to get current working directory")
+	}
+
+	basePath := filepath.Dir(filename)
+	SetFileLocationTesting(basePath)
```

Comment:
are we creating the manifests directory under the current directory? Can't we create the way it normally happens i.e. `~/.meshery/manifests`
I mean we only need the config location to be manually set so let's make other folders the way that would work in a real case scenario.

Let's improve it if it isn't working like that.